<launch>

<arg name="gui" default="False"/>


<param  name="robot_description"
        command="$(find xacro)/xacro --inorder '$(find open_base)/xacro/main.urdf.xacro'" />

<node name="driver_node_a" pkg="roboteq_python" type="roboteq_driver_node" output="screen">
    <rosparam command="load" file="$(find omnicone)/config/motors_a.yaml" />
</node>

<node name="driver_node_b" pkg="roboteq_python" type="roboteq_driver_node" output="screen">
    <rosparam command="load" file="$(find omnicone)/config/motors_b.yaml" />
</node>

<node name="kinematics_node" pkg="open_base" type="open_base_kinematics" output="screen"/>

<node name="vel_command_publisher" pkg="omnicone" type="vel_command_publisher" output="screen"/>

<node name="gps_publisher" pkg="omnicone" type="gps_publisher.py" output="screen"/>

<node name="pos_cmd" pkg="omnicone" type="rect_test_ccw.py" output="screen"/>

<node pkg="rosbag" type="record" name="robot_bag" args="-o /home/jacob/RectTestCCW /gps_pub /pos_estimate /robot_vel_goal" />

<!-- <node name="camera_preview" pkg="omnicone" type="camera_preview.py" if="$(arg camera_previ ew)"/> -->

</launch>
